src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Windows.targets

```diff
@@ -26,9 +26,16 @@ Copyright (c) .NET Foundation. All rights reserved.
   <!-- Set 7.0 as the TargetPlatformVersion for windows. -->
   <PropertyGroup Condition="'$(TargetPlatformIdentifier)' == 'windows' and '$(TargetPlatformVersion)' == ''" >
     <TargetPlatformVersion>7.0</TargetPlatformVersion>
-    <!-- Automatic generation of TargetPlatformMoniker and TargetPlatformDisplayName have already been evaluated so we need to hardcode these values. -->
-    <SupportedOSPlatform Condition="'$(SupportedOSPlatform)' == ''">$(TargetPlatformVersion)</SupportedOSPlatform>
-    <TargetPlatformMoniker>Windows,Version=$(TargetPlatformVersion)</TargetPlatformMoniker>
-    <TargetPlatformDisplayName>Windows $(TargetPlatformVersion)</TargetPlatformDisplayName>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetPlatformIdentifier)' == 'Windows'">
```

Comment:
Why does this need to be windows specific?

Reply:
We decided that the canonical property going forward should be SupportedOSPlatformVersion, so we're just supporting TargetPlatformVersion for Windows since it already exists.